cmd/build/v2/services_test.go

```diff
@@ -27,7 +28,9 @@ func TestAllServicesAlreadyBuilt(t *testing.T) {
 	fakeReg := test.NewFakeOktetoRegistry(nil)
 	bc := NewBuilder(nil, fakeReg)
 	alreadyBuilt := []string{}
-	fakeReg.AddImageByName(alreadyBuilt...)
+	if err := fakeReg.AddImageByName(alreadyBuilt...); err != nil {
+		oktetoLog.Infof("error adding image to registry: %s", err)
+	}
```

Comment:
Since it's a test, in this whole file we should `require.NoError` instead of logging the message